spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraSourceRelation.scala

```diff
@@ -168,4 +164,28 @@ object CassandraSourceRelation {
       writeConf = writeConf,
       sqlContext = sqlContext)
   }
+
+  /**
+   * Consolidate Cassandra conf settings in the order of table level -> keyspace level ->
+   * cluster level -> default. Use the first available setting
+   */
+  def consolidateConfs(sqlContext: SQLContext, tableRef: TableRef, cassandraConfs: Map[String, String]) : SparkConf = {
+    //Default settings
+    val conf = sqlContext.sparkContext.getConf.clone()
+    //Keyspace/Cluster level settings
+    val sqlConf = sqlContext.getAllConfs
+    for (prop <- DefaultSource.confProperties) {
```

Comment:
tiny change, I would have prop be called defaultProp so it is more clear in the following if statements